requirements.txt

```diff
@@ -66,3 +67,4 @@ wcwidth==0.1.7
 webencodings==0.5
 whitenoise==3.2.2
 wsgiref==0.1.2
+xhtml2pdf==0.2b1
```

Comment:
Are we sure production servers are set up to support these pdf libraries?

Reply:
Yup, should be fine.